maze/README.rst

```diff
@@ -0,0 +1,78 @@
+Maze Solver Demo
+================
+Getting the D-Wave Quantum Computer to solve a maze!
```

Comment:
We don't usually have such a happy sentence (strictly speaking this doesn't qualify as a sentence) in our documentation. That doesn't mean we shouldn't but give some thought to whether we should. In any case, if we keep it, we'd want "D-Wave quantum computer"

Reply:
I have fixed `Quantum Computer` -> `quantum computer`. Uncertain about the happiness level that we can convey.

Reply:
Unless there is reason to remove the exclamation mark, I will keep it for now.